tests/unit/workflows/test_workflows_tasks_actions.py

```diff
@@ -819,77 +818,6 @@ def test_populate_journal_coverage_does_nothing_if_no_journal_is_found(mock_repl
     assert 'journal_coverage' not in obj.extra_data
 
 
-@patch('inspirehep.modules.workflows.tasks.actions.get_document_in_workflow')
```

Comment:
why did you remove all these tests?

Reply:
They were unit tests, and they are bound to fail due to inclusion of the inspire matcher since there's no ElasticSearch instance in Unit tests.

Reply:
then they should become integration tests instead of unit, or mock the reference matching if they don't depend on it.